### ISSUE TITLE
Bug get surveys

### ISSUE THREAD

**Issue by amcastror** [SurveyGizmoOfficial/PHP_SurveyGizmo_Api#16]
Hi, I'm following the example to get surveys filtered by title, and I get the following error:

![screen shot 2018-02-21 at 7 35 56 pm](https://user-images.githubusercontent.com/3622318/36509457-945e8296-173e-11e8-8774-ac5fa8c083ac.png)

My code is very simple:

```php
require __DIR__ . '/../vendor/autoload.php';

try {
	\SurveyGizmo\SurveyGizmoAPI::auth("key", "secret");
} catch (Exception $e) {
	dump("Error Authenticating");
}

$filter = new \SurveyGizmo\Helpers\Filter();
$filter_item = new \SurveyGizmo\Helpers\FilterItem();
$filter_item->setField('title');
$filter_item->setOperator('=');
$filter_item->setCondition('test');
$filter->addFilterItem($filter_item);
$surveys = \SurveyGizmo\Resources\Survey::fetch($filter);

dump($surveys);
```

Am I doing something wrong? Thanks a lot.

PS: this are my dependencies:

```json
{
    "repositories": [
    ],
    "autoload": {
        "psr-4": {"": "src/"}
    },
    "require": {
  		"surveygizmo/surveygizmo-api": "^1.0",
  		"symfony/var-dumper": "~3.2"
    }
}
```

**Comment by amcastror** [comment]
I created [this PR](https://github.com/SurveyGizmoOfficial/PHP_SurveyGizmo_Api/compare/master...amcastror:patch-1) that fixes the issue.

#17

**Comment by SurveyGizmoOfficial** [comment]
Pull request merged, PHP 5.3 compatibility added, and release v1.0.2-stable published.

Thanks @amcastror!
